pkg/oci/config.go

```diff
@@ -79,23 +79,26 @@ type Config struct {
 	// VCNID is the OCID of the Virtual Cloud Network (VCN) within which the
 	// cluster resides.
 	VCNID string `yaml:"vcn"`
+
+	// Toggle managing routes
+	ManageRoutes bool `yaml:"manageRoutes"`
```

Comment:
You shouldn't need this flag/config option. The route controller is only going to run if the appropriate flags are set on the CCM. This causes unnecessary extra work on the end user.